app/screens/custom_status/custom_status.tsx

```diff
@@ -264,12 +282,15 @@ const CustomStatus = ({
                     status.duration = newStatus.duration;
                     status.expires_at = newExpiresAt;
                 }
+
                 const {error} = await updateCustomStatus(serverUrl, status);
                 if (error) {
+                    dismissModalAndKeyboard(isTablet);
```

Comment:
Curious: why this one (and 1-2 others) doesn't have option for componentId like some others?  Looking at dismissModal() function, it will close all open modals.  If that is the intention, then that's fine.  But if not, then this might close other modals unintendedly.

```
                    dismissModalAndKeyboard(isTablet, {componentId});
```

Reply:
thanks for bringing this up, unfortunately I am not very familiar with the code, my changes tried to be as less intrusive as possible so what I tried was to continue the logic that was before in the component. Maybe @larkox could provide more details on this.

Reply:
No strong opinion right now. I wonder what other modals may be open at the same time, but yeah, probably it is better to pass the id.

Reply:
ok, will send the change